RULES.md

```diff
@@ -156,6 +156,8 @@ Rules from the AWS NIST 800-53 Conformance Pack which could not be implemented.
 | -------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- | --------------------- |
 | [ec2-imdsv2-check](https://docs.aws.amazon.com/config/latest/developerguide/ec2-imdsv2-check.html)                   | The EC2 instance has IMDSV2 (instance metadata service version 2) enabled.                              | AC-6                  |
 | [efs-in-backup-plan](https://docs.aws.amazon.com/config/latest/developerguide/efs-in-backup-plan.html)               | EFS is in an AWS Backup plan.                                                                           | CP-9(b), CP-10, SI-12 |
+| [elasticache-redis-cluster-automatic-backup-check](https://docs.aws.amazon.com/config/latest/developerguide/elasticache-redis-cluster-automatic-backup-check.html) |
```

Comment:
I think we can actually check this one. Both ReplicationGroups and CacheClusters have the [snapshotretentionlimit](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticache-cache-cluster.html#cfn-elasticache-cachecluster-snapshotretentionlimit) properties which are used for configuring automatic backups